packages/manager/apps/procedures/src/pages/rgdp/rgdpForm/RGDPForm.constants.ts

```diff
@@ -0,0 +1,13 @@
+export const GDPRSubjectValues = [
+  'opposition_right',
+  'rectification_right',
+  'access_right',
+  'erasure_right',
+  'limitation_right',
+  'portability_right',
+  'payment_method_remove',
+  'other_request',
+];
+
+export const EmailRegex = /^[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\.)+[a-zA-Z0-9]{2}(?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$/;
```

Comment:
Are you sure for that regex ? I never see numbers in the tld part

Reply:
yep, i used Xander regex